src/components/layers-manager/CustomLayers.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Textarea } from '../form/Form';
+import { injectIntl } from 'react-intl';
+
+import withModal from '../ui/withModal';
+import Walkthrough from '../ui/Walkthrough';
+import Icon from '../ui/Icon';
+
+const WalkthroughModal = withModal(Walkthrough);
+
+const WALKTHROUGH_TEXTS = [
```

Comment:
For consistency these could go in a new walkthrough.js constants file, but as we also have language constants also, it could become confusing.